### PR TITLE
Closes #1560: Fix files downloading

### LINKED ISSUE #1560: On dataset page, only the first attempt of file download is succesful
The next attempt requires refreshing the page.

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/PrimefacesUtil.java [hunk 1]
```diff
@@ -14,6 +14,7 @@ public static void showDialog(String dialogWidgetVar) {
     }
     
     public static void showDialogAndResize(String dialogWidgetVar) {
+//        PrimeFaces.current().ajax().update(dialogWidgetVar);
```
Comment (wtr-icm) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/PrimefacesUtil.java]: to remove

File: dataverse-webapp/src/main/webapp/file.xhtml [hunk 2]
```diff
@@ -194,6 +194,10 @@
                             </div>
                         </div>
                     </div>
+                    <a id="currentDownloadRequestLink" href="#" target="_blank" rel="noopener noreferrer" download="" style="display: none;">
+                        download requested files
```
Comment (wtr-icm) [inline dataverse-webapp/src/main/webapp/file.xhtml:198]: Shouldn't be taken from bundle?

Comment (madryk) [reply dataverse-webapp/src/main/webapp/file.xhtml:198]: It's fully hidden from the user and it is intended to be this way all the time. It doesn't really matters if it is translated or not - so I decided not to add this to bundle to not create extra overhead in for example translation process (thinking about where it is used, what it does, etc.). I guess if we really want to add something from bundle then probably the best key would be: download=Download (as it is used either way in many places). What do you think?

Comment (wtr-icm) [reply dataverse-webapp/src/main/webapp/file.xhtml:198]: If it's hidden then it's ok. Though I wonder how will behave screen readers?

Comment (madryk) [reply dataverse-webapp/src/main/webapp/file.xhtml:198]: screen readers will omit things that are not visible (for example with `display: none;`).
Here is useful link https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/ that confirms this

**Review by wtr-icm — APPROVED** [review]

**Pull request merged**